src/locales/intl.test.js

```diff
@@ -0,0 +1,320 @@
+/* globals describe test expect beforeEach */
+import intl, {setIntlLocale} from './intl'
+import * as UTILS from '../../src/modules/utils'
+
+const EN_US_LOCALE = {
+  'localeIdentifier': 'en_US',
+  'decimalSeparator': '.',
+  'quotationBeginDelimiterKey': '“',
+  'quotationEndDelimiterKey': '”',
+  'currencySymbol': '$',
```

Comment:
We deal with currency symbols and currency codes in a different module. These should be removed

Reply:
Fixed